### ISSUE TITLE
Class Line count and function line count

### ISSUE THREAD

**Issue by Csardelacal** [phpmd/phpmd#107]
There seems to be a bug with the count of the length (in lines) of classes and functions. 

First of all, phpmd counts blank lines as a whole line, that means that any programmer that has the habit of separating variables at the beginning of a function with an extra blank line from the logic code and the if blocks too will get an error with even the shortest functions.

On a second note there is the error when counting inside a class. It includes the PHPDoc comments as code lines. This means that a 60 line class is count as 170 because most of the content is PHPDoc.

Thanks for the awesome work on the project!

**Comment by ravage84** [comment]
There is a property named ``ignore-whitespace`` to ignore the whitespace.
https://phpmd.org/rules/codesize.html

Another property for ignoring the doc blocks or all comments in general could be useful for some, I guess.

**Comment by stevecoug** [comment]
Would it make sense to have an option to count a multi-line string as one line of code?

**Comment by kylekatarnls** [comment]
Indeed, despite this one is very old the `ExcessiveClassLength` is not relevant today. The default value of 1000 lines is incredibly high probably because comments and blank lines are taken into account by default.

Indeed `ignore-whitespace` ignore lonely `{` or `}`, blank lines and comments. This option actually solves the issue here. In the next major/minor (to be discussed), ignoring those lines should be the default behavior, and so the default threshold should not be more than 500.
